src/main/java/uk/gov/hmcts/ccd/domain/service/search/elasticsearch/security/ElasticsearchCaseSearchRequestSecurity.java

```diff
@@ -62,4 +102,16 @@ private CaseSearchRequest createNewCaseSearchRequest(CaseSearchRequest caseSearc
         return new CaseSearchRequest(caseSearchRequest.getCaseTypeId(),
             new ElasticsearchRequest(searchRequestJsonNode));
     }
+
+    private CrossCaseTypeSearchRequest createNewCrossCaseTypeSearchRequest(CrossCaseTypeSearchRequest request,
+                                                                           String queryWithFilters) {
+        ObjectNode queryNode = objectMapperService.convertStringToObject(queryWithFilters, ObjectNode.class);
+
+        return new CrossCaseTypeSearchRequest.Builder()
+            .withCaseTypes(request.getCaseTypeIds())
+            .withSearchRequest(new ElasticsearchRequest(queryNode))
+            .withMultiCaseTypeSearch(request.isMultiCaseTypeSearch())
```

Comment:
i guess we may not need to populate `withMultiCaseTypeSearch` and `withSourceFilterAliasFields`